tests/test_utils.cpp

```diff
@@ -225,6 +238,32 @@ libcellml::ModelPtr createModelTwoComponentsWithOneVariableEach(const std::strin
     return model;
 }
 
+libcellml::ComponentPtr createImportInModel(const libcellml::ModelPtr &model,
+                                            const std::string &componentName,
+                                            const std::string &importSourceUrl,
+                                            const std::string &importReference)
+{
+    model->addComponent(libcellml::Component::create(componentName));
+    auto imp = libcellml::ImportSource::create();
+    model->component(componentName)->setImportSource(imp);
+    model->component(componentName)->importSource()->setUrl(importSourceUrl);
+    model->component(componentName)->setImportReference(importReference);
+    return model->component(componentName);
+}
+
+libcellml::ComponentPtr createImportInComponent(const libcellml::ComponentPtr &component,
+                                                const std::string &componentName,
+                                                const std::string &importSourceUrl,
+                                                const std::string &importReference)
+{
+    component->addComponent(libcellml::Component::create(componentName));
+    auto imp = libcellml::ImportSource::create();
+    component->component(componentName)->setImportSource(imp);
+    component->component(componentName)->importSource()->setUrl(importSourceUrl);
+    component->component(componentName)->setImportReference(importReference);
+    return component->component(componentName);
+}
```

Comment:
These methods are not being used anywhere, so should be removed.

Reply:
0ecc37d9